package-lock.json

```diff
@@ -74,7 +74,7 @@
                 "date-utils": "1.2.21",
```

Comment:
This file is going to cause merge conflict headaches and I'm not sure why Matt added it back. Can we remove it and add it back to the gitignore?